source/frontend/src/features/properties/map/project/ProjectContainerView.tsx

```diff
@@ -31,14 +68,41 @@ const ProjectContainerView: React.FC<IProjectContainerViewProps> = ({
       title={viewTitle}
       icon={<FaBriefcase className="mr-2 mb-2" size={32} />}
       header={<ProjectHeader project={project} />}
+      footer={
+        isEditing && (
```

Comment:
hmm, ok, so the user can get to this form when not editing?

Reply:
Tested again, haven't found a way to access the form when NOT editing.